Resources/views/elements/post-button.html.twig

```diff
@@ -1,6 +1,6 @@
 {% trans_default_domain 'EMSCoreBundle' %}{% if message is defined %}
 <div class="btn-group">
-  <button type="button" class="btn {% if class is defined %}{% else %}btn-sm{% endif %} btn-danger dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+  <button type="button" class="btn {% if class is defined %}{{ class }}{% else %}btn-sm btn-danger{% endif %} dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
To be verified with MDK, I think POST actions should always be btn-danger

Reply:
Agree, but it fix a bug

Reply:
Ok, so no longer the red look obligation for Post buttons?